### ISSUE TITLE
consider nameFrom: heading

### ISSUE THREAD

**Issue by pkra** [w3c/dpub-aria#50]
ARIA is gaining name-from-heading as an option for naming things, see https://github.com/w3c/aria/pull/1860.

Many DPUB roles could benefit from this feature, in particular those roles subclassing roles that will gain nameFrom:heading.

**Comment by mattgarrish** [comment]
Agree. A lot of publishers just expect a heading to label its role without having to add aria attributes.

But I'm going to mark these two issues with the next label until we get 1.1 out.
